lib/ably/realtime/channel.rb

```diff
@@ -179,15 +184,26 @@ def presence
         @presence
       end
 
-      # Return the message history of the channel.
+      # Return the message history of the channel
+      #
+      # Once attached to a channel, you can retrieve messages published on the channel before the
```

Comment:
`publish` calls queue_message https://github.com/ably/ably-ruby/blob/6a5f9607f18519ae0f58927d11a187371cda4091/lib/ably/realtime/channel.rb#L121 which in turn calls `attach` if it's not already https://github.com/ably/ably-ruby/blob/6a5f9607f18519ae0f58927d11a187371cda4091/lib/ably/realtime/channel.rb#L240-L244.

As for unsubscribe, my question was "does it implicitly detach too?". It's calling `super` but I believe if you did not specify anything in the method it should not, right?